lambdas/service/app.py

```diff
@@ -821,36 +825,76 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-@app.route('/manifest/files', methods=['GET'], cors=True)
-def start_manifest_generation():
-    """
-    Initiate and check status of a manifest generation job, returning a either a 301 or 302 response
-    redirecting to either the location of the manifest or a URL to re-check the status of the manifest job.
-
-    parameters:
-        - name: filters
-          in: query
-          type: string
-          description: Filters to be applied when generating the manifest
-        - name: format
-          in: query
-          type: string
-          description: The desired format of the output. Possible values are `compact` (the default) for a tab-separated
-          manifest and `terra.bdbag` for a manifest in the format documented `http://bd2k.ini.usc.edu/tools/bdbag/. The
-          latter is essentially a ZIP file containing two manifests: one for participants (aka Donors) and one for
-          samples (aka specimens). The format of the manifests inside the BDBag is documented here:
-          https://software.broadinstitute.org/firecloud/documentation/article?id=10954
-        - name: token
-          in: query
-          type: string
-          description: Reserved. Do not pass explicitly.
+manifest_description = unwrap('''
+    <p>The desired format of the output.
+    <ul>
+    <li> `compact` (the default) for a compact tab-separated manifest
+    <li> `full` for a full tab-separated manifest
+    <li> `terra.bdbag` for a manifest in the format documented
+    [http://bd2k.ini.usc.edu/tools/bdbag/]. This provides a ZIP file containing
+    two manifests: one for participants (aka Donors) and one for samples (aka
+    specimens). The format of the manifests inside the BDBag is documented here:
+    [https://software.broadinstitute.org/firecloud/documentation/article?id=10954]
+    </ul>
+''')
 
-    :return: If the manifest generation has been started or is still ongoing, the response will have a
-    301 status and will redirect to a URL that will get a recheck the status of the manifest.
 
-    If the manifest generation is done and the manifest is ready to be downloaded, the response will
-    have a 302 status and will redirect to the URL of the manifest.
-    """
+@app.route('/manifest/files', methods=['GET'], cors=True, path_spec={
```

Comment:
Removed redundancy in the documentation.

Reply:
`start_manifest_generation` and `start_manifest_generation_fetch` both just call `handle_manifest_generation_request` (but return different forms of responses). This means they use the same code to parse the filters parameter. This implies that either both should have `file=` in the description, or neither should. In any case the descriptions of parameters will be the same.

It looks like all of the parameters (and therefore the entire `path_spec`) in the manifest endpoints should be the same, and therefore extracted out into its own variable. Since the variables you created for `manifest_filters`, `manifest_param_token` and `manifest_param_format` would only be used in once, in this _new_ variable, they should probably just be inlined.